modules/sagemaker/sagemaker-templates-service-catalog/templates/multi_account_basic/product_stack.py

```diff
@@ -151,25 +238,20 @@ def __init__(
             )
         )
 
-        # cross account model registry resource policy
         model_package_group_name = f"{sagemaker_project_name}-{sagemaker_project_id}"
-        model_package_arn = (
```

Comment:
Minor - looks like this has overwritten changes from the recent PR https://github.com/awslabs/mlops-modules/pull/33 ... need to rebase or merge main